app/helpers/mediums_helper.rb

```diff
@@ -1,3 +1,9 @@
 module MediumsHelper
   include PtablesHelper
+
+  def required_nfs_list
+    Hash[Operatingsystem.families.map do |key|
+      [key, key.constantize.require_nfs_access_to_medium]
+    end].to_json
+  end
```

Comment:
Why not just return a list of all families requiring it and check if the selected family is in it or not?